src/prefect/indicators/infrastructure/i1.py

```diff
@@ -19,7 +19,21 @@
 
 from ..conf import settings
 from ..models import Indicator, IndicatorPeriod, Level
+from ..util import params_level, table_level
 
+VALUES_FOR_TARGETS_TEMPLATE = """
+        SELECT
+            COUNT(DISTINCT PointDeCharge.id_pdc_itinerance) AS value,
```

Comment:
```suggestion
            COUNT(DISTINCT PointDeCharge.id_pdc_itinerance) AS num_poc,
```

Reply:
-> value retention to avoid later modification (see discussion)